### ISSUE TITLE
Update on crates.io

### ISSUE THREAD

**Issue by maddin77** [zeyla/urbandictionary.rs#2]
```
    Updating registry `https://github.com/rust-lang/crates.io-index`
error: no matching version `^0.2` found for package `urbandictionary`
location searched: registry https://github.com/rust-lang/crates.io-index
versions found: 0.1.1, 0.1.0
```
Can you update crates.io to `0.2`?

**Comment by zeyla** [comment]
I will be able to get to this in the next day or so

**Comment by zeyla** [comment]
Hey, sorry about the wait. v0.2 is now on crates.io.

A little bit of an (annoying) story about why it took me so long to get to this:

I had a previous GitHub account that I published most of my current project on, but deleted it. This account is newer and has the same name. I kept my old cargo key for package management though. When I went to add this GitHub account as an owner of the crates, it said it already was. So I needed to find time to rename this account and transfer all the crates over in one go, add this account as an owner from the old key, rename the account, and then use the new key to remove the old account.

As for the library, I have a few things on my mental to-do list, a v0.3 should be out in a couple weeks if I find the time.
